python/pyspark/sql/functions.py

```diff
@@ -14394,6 +14394,260 @@ def nvl2(col1: "ColumnOrName", col2: "ColumnOrName", col3: "ColumnOrName") -> Co
     return _invoke_function_over_columns("nvl2", col1, col2, col3)
 
 
+@try_remote_functions
```

Comment:
Suggest placing the function in the appropriate location of the corresponding file

Reply:
relocate the functions by group, thanks.